File: week8/PureAbs.java
interface Interface {
    void AbstractMethod();
}

public class PureAbs implements Interface {
    public void AbstractMethod() {
        System.out.println("This method implements the abstract method of the interface.");
    }

    public static void main(String[] args) {
        PureAbs obj = new PureAbs();
        obj.AbstractMethod();
    }

}

File: week8/PureAbs.cpp
#include<iostream>
using namespace std;

class AbstractClass {
public:
    virtual void abstractMethod() = 0;
};

class PureAbs : public AbstractClass {
public:
    void abstractMethod() override {
        cout << "This is a pure virtual function implemented in a PureAbs class" << endl;
    }
};
int main() {
    PureAbs obj;
    obj.abstractMethod();
    return 0;
}
